src/components/functionalComponent.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+const helloWorldComponent = (props) => (<h1>Hello World in functional component</h1>);
```

Comment:
Functional components still need to be started with upper case letter. Also you can omit props here, but leave it function, I mean `() => (<h1>Hello World in functional component</h1>);`

Reply:
done